### ISSUE TITLE
Creation of .nfo file

### ISSUE THREAD

**Issue by imolb** [fnep/mtv_dl#14]
Would be great if a .nfo file for each downloaded movie is created.
This should be compatible for Kodi https://kodi.wiki/view/NFO_files#Metadata_nfo

**Comment by fnep** [comment]
Cool Idea :)

I have no time these days but will note it down for later this month.

**Comment by fnep** [comment]
I had a first look and have some questions. Did you have contact with .nfo files already? 

They support movies and tv shows (and music stuff). The problem here is, that mtv_dl does not know if a download is a "serial" tv show or a movie. Also mtv_dl is missing most of the information required for tv shows. The only way we could use this is to provide a movie-.nfo for all downloads.

Would this confuse Kodi? What information would you expect to get with this file (more than title and description)?

**Comment by fnep** [comment]
Please have a look to the 'nfo' branch for how this could look like. Here i used <movie> and tried to guess the assignment.

**Comment by imolb** [comment]
Looking at the code seems exactly to fit my issue! 
Unfortunately testing needs to wait three weeks. I'll report than.

**Comment by fnep** [comment]
Sure, no problem. I will wait :)

**Comment by imolb** [comment]
I did some testing and the .nfo files are generated as expected.
Unfortunately Kodi ignores these files.
I've created some .nfo files manually (with the same xml tags and contents) and these are used by Kodi. I tried to approximate the manual written ones to the generated ones, to figure out, what is stopping Kodi from using the generated ones (maybe something with encoding, newlines, ids, ...). But no success yet. I'll need some more time and report than.
